hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/snapshot/ExportSnapshot.java

```diff
@@ -1082,13 +1086,16 @@ public int doWork() throws IOException {
     }
 
     // Write a new .snapshotinfo if the target name is different from the source name
```

Comment:
Better also change this comment to mention that if we want to reset ttl then we also need to write a new .snapshotinfo?